### PR TITLE
Update to 3.2.0

### PR DESCRIPTION
**Pull request by nervo**
\o/

### CODE REVIEW

**Comment by NicolasDievart** [comment]
👍

**Comment by ogizanagi** [comment]
🎉

File: app/config/config.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ framework:
     fragments:       ~
     http_method_override: true
     assets: ~
+    php_errors:
+        log: true
+
```
Comment (benji07) [inline app/config/config.yml]: 2 empty lines ?

Comment (ogizanagi) [inline app/config/config.yml]: It's from upstream https://github.com/symfony/symfony-standard/blob/master/app/config/config.yml#L34-L37

:trollface:

Comment (chalasr) [inline app/config/config.yml]: https://github.com/symfony/symfony-standard/pull/1027 :)

Comment (nervo) [inline app/config/config.yml]: It's upstream...

Comment (ogizanagi) [inline app/config/config.yml]: Upstream fix has been merged 😄

**Pull request merged**